src/lib_gui/qt/project_wizard/QtProjectWizard.cpp

```diff
@@ -1012,12 +1013,12 @@ void QtProjectWizard::createProject()
 	{
 		MessageStatus(L"Unable to save project to location: " + path.wstr()).dispatch();
 
-		QMessageBox msgBox(this);
```

Comment:
Please add the `(this)` again. Got probably lost during rebase.

Reply:
ah okay.